templates/medium/cxf-jaxrs/medium-fuse-cxf-jaxrs-spring/pom.xml

```diff
@@ -0,0 +1,217 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.apache.cxf.archetype</groupId>
+    <artifactId>cxf-jaxrs-service</artifactId>
+    <version>3.0.4.redhat-620133</version>
+    <packaging>war</packaging>
+    <name>Simple CXF JAX-RS webapp service using spring configuration</name>
+    <description>Simple CXF JAX-RS webapp service using spring configuration</description>
+    <properties>
+        <jackson.version>1.8.6</jackson.version>
```

Comment:
current jackson version is 2.8.0. I think that we can try to update it.
Or is it to be aligned with Fuse version?